doc/Programming-guide/Your-first-Kaa-application/attach/demo-cpp/KaaDemo.cpp

```diff
@@ -56,6 +56,11 @@ class ConfigurationCollection : public kaa::IConfigurationReceiver {
     {
         // Run the Kaa endpoint.
         kaaClient_->start();
+        // Read default sample period
+        samplePeriod_ = kaaClient_->getConfiguration().samplePeriod;
+        std::cout << "Set default sample period: " << samplePeriod_<< std::endl;
```

Comment:
what do you mean "Could you also split the initialization _(samplePeriod_ = 0) ?" ?